src/data/testnet/celo-alfajores-tokens-info.json

```diff
@@ -3,6 +3,14 @@
     "address": "0x00be915b9dcf56a3cbe739d9b9c202ca692409ec",
     "decimals": 18,
     "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/UBE.png",
+    "name": "Old Ubeswap",
+    "symbol": "old-UBE",
+    "infoUrl": "https://www.coingecko.com/en/coins/ubeswap"
+  },
+  {
+    "address": "0x71e26d0e519d14591b9de9a0fe9513a398101490",
```

Comment:
@MuckT looks like it's an EOA on [Alfajores](https://explorer.celo.org/alfajores/address/0x71e26d0E519D14591b9dE9a0fE9513A398101490)

and it's probably causing a [CI failure](https://github.com/valora-inc/blockchain-api/actions/runs/8519832511/job/23334810366?pr=448) on `blockchain-api`.

Reply:
Opted to use the old address on Alfajores in https://github.com/valora-inc/address-metadata/pull/442.